npm/extensions.bzl

```diff
@@ -62,9 +93,10 @@ def _extension_impl(module_ctx):
                     custom_postinstall = i.custom_postinstall,
                     deps = i.deps,
                     integrity = i.integrity,
-                    lifecycle_hooks = i.lifecycle_hooks,
+                    lifecycle_hooks = i.run_lifecycle_hooks if i.run_lifecycle_hooks and i.lifecycle_hooks else [],
```

Comment:
This was the issue. I had to mimic what was being put into `{maybe_lifecycle_hooks}` which had this conditional.

Reply:
👍